domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevbestiller/impl/DokgenLanseringTjeneste.java

```diff
@@ -15,7 +15,8 @@
             DokumentMalType.IKKE_SØKT,
             DokumentMalType.VARSEL_OM_REVURDERING,
             DokumentMalType.INGEN_ENDRING,
-            DokumentMalType.INFOBREV_TIL_ANNEN_FORELDER);
+            DokumentMalType.INFOBREV_TIL_ANNEN_FORELDER,
```

Comment:
Bytter du om rekkefølgen på de to siste så den blir lik med DEV - da er det lettere å se hva som er lansert og ikke

Reply:
done